cmd/ostracon/commands/init.go

```diff
@@ -73,6 +73,10 @@ func initFilesWithConfig(config *cfg.Config) error {
 		logger.Info("Found genesis file", "path", genFile)
 	} else {
 		genDoc := types.GenesisDoc{
+			// FIXME: It shouldn't be generated automatically, and the config and genDoc chainIDs should be the same.
```

Comment:
I had mistakenly thought that the `BaseConfig.chainID` field was a duplicate setting also described in `config/config.toml` independent of `genDoc`, even though it has the same semantics. I've confirmed that this field is not subject to serialization/deserialization because it doesn't have a `mapstructure` and is, therefore, volatile information rather than persistent information stored in `config.toml`. Thus, I'll remove this comment.

Upon my exam, it appears that `BaseConfig.chainID` isn't used outside of this location and tests. I guess we need to discuss whether it's really a necessary field, but that should be done outside of this PR.